activiti-core/activiti-spring-boot-starter/src/test/java/org/activiti/spring/boot/tasks/TaskRuntimeCandidatesTest.java

```diff
@@ -166,8 +160,22 @@ public void should_addAndRemoveCandidateGroup() {
         groupCandidates = taskRuntime.groupCandidates(createTask.getId());
         assertThat(groupCandidates).isNotNull();
         assertThat(groupCandidates.size()).isEqualTo(0);
-    }
 
+        taskRuntime.addCandidateGroups(TaskPayloadBuilder
+                                               .addCandidateGroups()
+                                               .withTaskId(createTask.getId())
+                                               .withCandidateGroup("test")
+                                               .build());
+
+        task = taskRuntime.task(createTask.getId());
+        groupCandidatesOnTask = task.getCandidateGroups();
+        assertThat(groupCandidatesOnTask).isNotNull();
+        assertThat(groupCandidatesOnTask.size()).isEqualTo(1);
```

Comment:
Tip: you can use the shortcut `assertThat(groupCandidatesOnTask).hasSize(1)`. This will also check that `groupCandidatesOnTask` is not null, so you can remove line `172`.